code/modules/paperwork/paper_premade.dm

```diff
@@ -9,6 +9,30 @@
 /obj/item/paper/fluff/shuttles/daniel
 	info = "i love daniel<br>daniel is my best friend<br><br>you are tearing me apart elise"
 
+/obj/item/paper/fluff/trickwines_4_brewers
+	name = "trickwine 4 brewers";
+	info = {"trickwine 4 brewers
+	<br>### Breakaway Flasks
+	<br>The primary container for trickwines as all wines currently only can be made inside of one.
+	<br>Normal bottles do not splash targets when thrown, so breakaway flasks allow you hit the target with bonus damage from the glass shattering violently on the target.
+	<br>They can be created with 5 glass and 1 plasma sheet which is more expensive then most containers
+
+	<br><br>### Ashwine
+	<br>The primary trick wine produced by the SRM, as it is used in its religious practices.
+	<br>On drink: A mild hallucinegenic effect which boosts mood
+	<br>On throw: A potent hallucigenic which causes screen shake, on a simplemob it should stun/paralyze
```

Comment:
Simplemob is excessively OOC language. These could all use a go through to be more... IC